contracts/test/mocks/MockRevertingERC20.sol

```diff
@@ -58,6 +60,7 @@ contract MockRevertingERC20 {
     }
 
     function _transfer(address owner_, address recipient_, uint256 amount_) internal {
+        require(recipient_ != revertingDestination, "INVALID");
```

Comment:
I think we should just replace this function with `require(false, "MOCK_TRANSFER_REVERT");`

Reply:
And then we can get rid of the `revertingDestination`